lib/stellar/account_signer.rb

```diff
@@ -0,0 +1,24 @@
+module Stellar
+  class AccountSigner
+    include Contracts
+
+    attr_reader :address
```

Comment:
I think this is the right word in this context, since `Stellar::KeyPair.from_address` and Stellar::Account.from_address take a string while `Stellar::KeyPair.from_public_key` takes encoded bytes. There's also a `Stellar::PublicKey` class.

Reply:
I agree in the context of an account's identifier, that is an address, but a signer is not an account. Those keys might be non-accounts. Horizon refers to the signer as having a `key`, and since this type is just a wrapper around Horizon data it's not obvious to me why we should rename things that Horizon has given names to. Thoughts @ire-and-curses ?